### PR TITLE
test: Avoid using obsolete GetRect methods

### LINKED ISSUE #10059: Remove current usages of `GetRect` in UI Tests
These obsolete methods should not be used:

https://github.com/unoplatform/uno/blob/9ae73a42303894d084634ea018457a3974de3e8c/src/SamplesApp/SamplesApp.UITests/Extensions/QueryExtensions.cs#L83-L100

and they should be removed.

### PR DESCRIPTION
**Pull request by Youssef1313**
GitHub Issue (If applicable): closes #10059

## PR Type

What kind of change does this PR introduce?

## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] Docs have been added/updated which fit [documentation template](https://github.com/unoplatform/uno/blob/master/doc/.feature-template.md) (for bug fixes / features)
- [ ] [Unit Tests and/or UI Tests](https://github.com/unoplatform/uno/blob/master/doc/articles/uno-development/working-with-the-samples-apps.md) for the changes have been added (for bug fixes / features) (if applicable)
- [ ] Validated PR `Screenshots Compare Test Run` results.
- [ ] Contains **NO** breaking changes
- [x] Associated with an issue (GitHub or internal) and uses the [automatic close keywords](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue).
- [x] Commits must be following the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/#summary) specification.

## Other information

Internal Issue (If applicable):

### CODE REVIEW

**Review by Youssef1313 — COMMENTED** [review]
@jeromelaban @carldebilly

**Review by iury-kc — APPROVED** [review]

**Review by jeromelaban — APPROVED** [review]

**Comment by amelield** [comment]
Is there a difference in usage between GetRect() and GetPhysicalRect(), beside one of them being obsolete?

**Comment by Youssef1313** [comment]
Yes @amelield 

https://github.com/unoplatform/uno/blob/90cc75873d1d6d46899245f17ece8d232015aba6/src/SamplesApp/SamplesApp.UITests/Extensions/QueryExtensions.cs#L134-L142

File: src/SamplesApp/SamplesApp.UITests/Windows_UI_Xaml_Controls/ContentDialog/UnoSamples_Tests.ContentDialog.cs [hunk 1]
```diff
@@ -398,7 +398,7 @@ public void ContentDialog_Simple_NotLightDismissible()
 			var dialogOpenedScreenshot = CurrentTestTakeScreenShot("1 ContentDialog Opened");
 
 			// tapping outside of dialog
-			var dialogRect = _app.GetRect(dialogSpace);
+			var dialogRect = _app.GetPhysicalRect(dialogSpace);
```
Comment (Youssef1313) [inline src/SamplesApp/SamplesApp.UITests/Windows_UI_Xaml_Controls/ContentDialog/UnoSamples_Tests.ContentDialog.cs:401]: This should probably be `GetLogicalRect`

File: src/SamplesApp/SamplesApp.UITests/Windows_UI_Xaml_Controls/ToolTipTests/ToolTip_Tests.cs [hunk 2]
```diff
@@ -62,10 +62,10 @@ public void ToolTip_Large_Text()
 			const string BorderInsideToolTip = "BorderInsideToolTip";
 
 			_app.WaitForElement(ButtonWithTooltip);
-			var buttonRect = _app.GetRect(ButtonWithTooltip);
+			var buttonRect = _app.GetPhysicalRect(ButtonWithTooltip);
 
-			var upperRect = _app.GetRect("UpperLocator");
-			var lowerRect = _app.GetRect("LowerLocator");
+			var upperRect = _app.GetPhysicalRect("UpperLocator");
+			var lowerRect = _app.GetPhysicalRect("LowerLocator");
```
Comment (Youssef1313) [inline src/SamplesApp/SamplesApp.UITests/Windows_UI_Xaml_Controls/ToolTipTests/ToolTip_Tests.cs:69]: Should probably be GetLogicalRect.

**Pull request merged**